packages/form-core/src/validate/ValidateMixin.js

```diff
@@ -200,6 +208,11 @@ export const ValidateMixinImplementation = superclass =>
       super.firstUpdated(changedProperties);
       this.__validateInitialized = true;
       this.validate();
+      if (this._repropagationRole !== 'child') {
+        this.addEventListener('model-value-changed', () => {
+          this.__childModelValueChanged = true;
```

Comment:
```suggestion
        this.validate({ clearCurrentResult: true });
```

Then we don't need the __childModelValueChanged lit property right?

Reply:
Yeah, that's what I suggested initially to Ben Amor as well :) But I thought it would be better to reduce the amount of sync handlers in the form and let it 'breath' 1 microtask

Reply:
For the reusability of mixins it might even be better to listen for this in FormControlMixin and do a requetsUpdate for modelValue (for groups that only have setters/getters for children)...
I think we can optimize this later (for now it's  a private value that can easiliy be refactored/improved upon)

Reply:
True! I also felt like maybe ValidateMixin wouldn't be the most logical place for the prop indeed